src/pages/project/project_wizard/ProjectForm.tsx

```diff
@@ -84,7 +85,10 @@ export const ProjectForm = ({ project }: ProjectFormProps) => {
       <SkipLink href="#form">{t('common.skip_to_schema')}</SkipLink>
       <Formik initialValues={project} validationSchema={basicProjectValidationSchema} onSubmit={submitProjectForm}>
         <Form noValidate>
-          <PageHeader variant="h1">{project.title}</PageHeader>
+          <PageHeader>
+            <ProjectIconHeader projectStatus={projectWithId ? projectWithId.status : undefined} />
+            <Typography variant={'h1'}>{project.title}</Typography>
```

Comment:
Ser vi bruker `TruncatableTypography` i den gamle `PageHeader`, så kan være det hadde vært lurt her også, for å unngå at den tar veldig mye plass ved lange prosjektnavn. Trolig mest relevant på mobil, men vet ikke hvor lange navn vi reelt kan forvente da..

Reply:
Godt poeng!